zerver/lib/export.py

```diff
@@ -1271,7 +1272,8 @@ def re_map_foreign_keys(data_table: List[Record],
                         field_name: Field,
                         related_table: TableName,
                         verbose: bool=False,
-                        id_field: bool=False) -> None:
+                        id_field: bool=False,
+                        recipient_field: bool=False) -> None:
```

Comment:
'recipient_field' is added as a bool variable in the function
'update_id_map' to update the recipient foreign keys.

Recipient Foreign Key is equal to the UserProfile ID, if the
type is 1, and the same is equal to Stream ID, if the type is 2.
Hence a check is added in the 'update_id_map' field for this.

Reply:
I suspect we need to new logic for re-mapping huddle foreign keys as well.  

You can test this with group PMs.